package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "privacy-grade",
-  "version": "0.9.1",
+  "name": "@duckduckgo/privacy-grade",
+  "version": "1.0.0",
```

Comment:
Scoping to `@duckduckgo/privacy-grade` so anyone in our org can do NPM releases if necessary.